test/junit/scala/collection/SeqTest.scala

```diff
@@ -90,4 +90,11 @@ class SeqTest {
     assert(s.lengthIs > 2)
     assert(s.lengthIs != 2)
   }
+
+  @Test
+  def testEndsWith(): Unit = {
+    assertEquals((0 until 0).endsWith(List(-4, -3, -2, -1)), false )
+    assertEquals((-8 until -1).endsWith((-8 until -1).takeRight(1)), true)
+    assertEquals((0 until 0).endsWith(0 until 0), true)
```

Comment:
Prefer `assertTrue` and `assertFalse`.

Reply:
This is now updated in my latest commit to this PR.